resource-managers/mesos/src/main/scala/org/apache/spark/deploy/rest/mesos/MesosRestServer.scala

```diff
@@ -82,6 +82,12 @@ private[mesos] class MesosSubmitRequestServlet(
     val mainClass = Option(request.mainClass).getOrElse {
       throw new SubmitRestMissingFieldException("Main class is missing.")
     }
+    val appArgs = Option(request.appArgs).getOrElse {
+      throw new SubmitRestMissingFieldException("Application arguments are missing.")
```

Comment:
Nit: maybe put the fields here also? Something like `Application arguments ("appArgs") are missing`. Similar below.

Reply:
Done @ArtRand